lib/src/ui/slider_captcha.dart

```diff
@@ -216,7 +225,7 @@ class _SliderCaptchaState extends State<SliderCaptcha>
 
   @override
   void didChangeDependencies() {
-    WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
+    WidgetsBinding.instance?.addPostFrameCallback((timeStamp) {

```

Comment:
Hi @Meghatronics.  
It seems that this synstax belongs to verison before 3.0.
The latest version of fluttter is 3.3.8 and its systax won't have "?" after WidgetsBinding.instance.

If you want it to work well on both version 2.x.x. and 3.x.x please follow the document:
https://docs.flutter.dev/development/tools/sdk/release-notes/release-notes-3.0.0
![Screen Shot 2022-11-21 at 12 42 42](https://user-images.githubusercontent.com/49116308/202974115-b1cb7ec6-8946-4e7c-a80c-cca66f075715.png)

Reply:
Hello @BrianTV98  
Thank you for reviewing my PR so quickly. 

I've pushed a commit attending to this.